frontend/src/metabase/redux/embed/embed.ts

```diff
@@ -112,11 +132,6 @@ function excludeNonInteractiveEmbeddingOptions(
 function normalizeEntityTypes(
   searchOptions: Partial<InteractiveEmbeddingOptions>,
 ): Partial<InteractiveEmbeddingOptions> {
-  const ALLOWED_ENTITY_TYPES: InteractiveEmbeddingOptions["entity_types"] = [
```

Comment:
Remove any `entityTypes` value normalization to embedding data picker slice instead.

What's left here is to normalize the array value from operating on the search parameters.